backend/inspirehep/matcher/utils.py

```diff
@@ -77,3 +80,70 @@ def copy_file(src_file, dst_file, buffer_size=io.DEFAULT_BUFFER_SIZE):
     while next_chunk:
         dst_file.write(next_chunk)
         next_chunk = src_file.read(buffer_size)
+
+
+def create_journal_dict():
+    """
+    Returns a dictionary that is populated with refextracts's journal KB from the database.
+
+    Uses two raw DB queries that use syntax specific to PostgreSQL to generate
+    a dictionary in the format that refextract expects as following::
+
+        { SOURCE: DESTINATION }
+
+    which represents that ``SOURCE`` is translated to ``DESTINATION`` when found.
+
+    Note that refextract expects ``SOURCE`` to be normalized, which means removing
+    all non alphanumeric characters, collapsing all contiguous whitespace to one
+    space and uppercasing the resulting string.
+    """
+
+    titles_query = db.session.execute(
+        """
+        SELECT
+            r.json -> 'short_title' AS short_title,
+            r.json -> 'journal_title' -> 'title' AS journal_title
+        FROM
+            records_metadata AS r
+        WHERE
+            (r.json -> '_collections')::jsonb ? 'Journals'
+    """
+    )
+
+    title_variants_query = db.session.execute(
+        """
+        SELECT
+            r.json -> 'short_title' AS short_title,
+            jsonb_array_elements((r.json -> 'title_variants')::jsonb) AS title_variant
+        FROM
+            records_metadata AS r
+        WHERE
+            (r.json -> '_collections')::jsonb ? 'Journals'
+    """
+    )
```

Comment:
no, you should use the ORM for this, in order to benefit from caching of repeated queries (see story).